src/network/networkd-manager.c

```diff
@@ -690,6 +694,11 @@ int manager_start(Manager *m) {
 
         assert(m);
 
+        sysctl_add_monitor(m);
+        r = drop_capability(CAP_SYS_ADMIN);
+        if (r < 0)
+                log_warning_errno(r, "Failed to drop CAP_SYS_ADMIN: %m");
```

Comment:
suffix with `, ignoring: %m`.

Also please add comment here that makes clear that we only kept CAP_SYS_ADMIN until here for the bpf stuff.

btw, which operation precisely requires CAP_SYS_ADMIN? in the comment also document that, please.